src/style.css

```
@@ -1,3 +1,112 @@
 body {
   margin: 0;
+  font-family: "Roboto", "Helvetica", sans-serif;
+  --color-blue: rgba(0, 172, 220, 1);
+  --color-black: rgba(6, 11, 53, 1);
+  --color-grey: rgba(97, 96, 112, 1);
+
+}
+
+.card {
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  width: 200px;
+  margin: auto;
+  padding: 33px 17px 17px;
+  box-sizing: border-box;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 35px auto 0;
+  color: var(--color-black);
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-grey);
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  display: block;
+  margin: auto;
+}
+
+.card__stars {
```

1) add margin-top
2) rename
```suggestion
.card__rating {
```